core/libdeno.rs

```diff
@@ -19,16 +19,9 @@ pub struct isolate {
   _unused: [u8; 0],
 }
 
-/// If "alloc_ptr" is not null, this type represents a buffer which is created
-/// in C side, and then passed to Rust side by `deno_recv_cb`. Finally it should
-/// be moved back to C side by `deno_respond`. If it is not passed to
-/// `deno_respond` in the end, it will be leaked.
-///
-/// If "alloc_ptr" is null, this type represents a borrowed slice.
+/// This type represents a borrowed slice.
```

Comment:
yesss

Reply:
If you're replying to this comment, unfortunately this is more or less true on the Rust side, but not entirely on the C++ side.
For example, deno_buf is also used for the `shared_`, the shared buffer, where it represents an owned buffer.
I hope to clean that up at some point but let's do one thing at a time.